src/auth/user-record.ts

```diff
@@ -35,6 +35,29 @@ function parseDate(time: any): string {
   return null;
 }
 
+/** Parameters for create user operation */
+export type CreateRequest = {
```

Comment:
Wouldn't it be better to use interfaces:
export interface UpdateRequest {
  displayName?: string;
  email?: string;
  ...
}
export interface CreateRequest extends UpdateRequest {
   uid?: string;
}